routes/api/users.js

```diff
@@ -1,9 +1,12 @@
 import { Router } from 'express';
+import bcrypt from 'bcrypt';
+import checkEmail from '../../helpers/checkEmail';
```

Comment:
Instead of having a separate helper class for this, we can have it in the utilities helper class.